appu.py

```diff
@@ -37,3 +38,6 @@
 )
 
 logger.warning("Done! File {} generated correctly".format(cfg['final_file']))
+
+if upload_file(cfg['final_file'], cfg['podcast_bucket']):
+    logger.warning("Episode uploaded to {}".format(cfg['podcast_bucket']))
```

Comment:
¿warning? This could be in info level IMHO

Reply:
:+1: As well as all the other messages in this file. I will add another commit fixing these, too.

Reply:
Fixed!